R/create_task_makefile.R

```diff
@@ -130,7 +136,7 @@ create_task_makefile <- function(
     if (('remake' %:::% 'target_is_file')(target, file_extensions = pos_extensions)){
       paste0("'", target, "'")
     } else {
-      target
+      if(tickquote_combinee_objects) paste0("`", target, "`") else target
```

Comment:
well that was pretty easy/clean

Reply:
yeah, i should have looked before worrying/complaining =)

Reply:
did you want `tickquote_combinee_objects` or `tickquote_combine[r]_objects`?

Reply:
i intended `combinee` because it's referring to objects that get combined. What's your thinking?

Reply:
haha, sounds good. I thought maybe it was a typo, but I get it now.